scripts/_setup-macos.sh

```diff
@@ -35,7 +35,6 @@ set -x # Print commands that are executed.
 FB_OS_VERSION=v2021.05.10.00
 NPROC=$(sysctl -n hw.physicalcpu)
 COMPILER_FLAGS="-mavx2 -mfma -mavx -mf16c -masm=intel -mlzcnt"
-DEPENDENCY_DIR=${DEPENDENCY_DIR:-$(pwd)}
```

Comment:
If we define `DEPENDENCY_DIR` before sourcing the `_setup-macos.sh`, do we still need to remove this line? ;)

Reply:
Why we need to keep it as `DEPENDENCY_DIR` has been declared.

Reply:
@ejguan In theory to keep the file same as https://github.com/facebookincubator/velox/blob/78159a995038e38576c354348023f50cc49a4864/scripts/setup-macos.sh so in the future it's easy to upgrade if Velox update's the `setup-macos.sh` ?

https://github.com/facebookresearch/torcharrow/blob/3fd191043989d2130d7df93535d795bbd64c3313/scripts/_setup-macos.sh#L16-L18

Reply:
Then, why not delete this file and call the script from velox directly?

Reply:
Guess we can also do this? -- but we don't have Mac CI yet so we don't know whether our Mac build might break when update Velox submodule.